### DISCUSSION TITLE
MALICIOUS USE OF SOMEONE ELSE'S AVATAR (someone is doing it to me right now!)

### DISCUSSION THREAD

**Discussion by EthanandT** [community/community#18533] [Other Feature Feedback, Questions, & Ideas]
How do you prevent people from maliciously creating another persons avatar and then doing nefarious things with it?

**Comment by Zerotask** [comment]
You can't really do anything against it except reporting that user to GitHub

**Comment by Sold77** [comment]
Prosecute the person

Den tis 14 juni 2022 kl 00:30 skrev EthanandT ***@***.***>:

> How do you prevent people from maliciously creating another persons avatar
> and then doing nefarious things with it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/github-community/discussions/18533>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASNF3WX7LICRUXIU3DKOVXDVO6Y7TANCNFSM5YVWHGHA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Mvh
Tommy
070 - 344 92 74

**Comment by MathiasReker** [comment]
Report the user to GitHub. There is a report feature on every profile.

**Comment by ifsdk** [comment]
If you worry about auto generated pixel icons, the some sort of solution would be to return their old style as opposite to circled avatars (and so uploaded fake pixel icons) as I insist in #28237

**Comment by radar-blip** [comment]
Like others have said, report the imposter avatar. 
Your Github created avatar is auto-generated and unique to your github username, so staff will have no problem identifying the imposter.
